cli/src/pcluster/config/cluster_config.py

```diff
@@ -366,6 +366,7 @@ def __init__(
         self.auto_import_policy = Resource.init_param(auto_import_policy)
         self.drive_cache_type = Resource.init_param(drive_cache_type)
         self.file_system_type = LUSTRE
+        self.file_system_type_version = "2.12" if backup_id is None else None
```

Comment:
What if file_system_id is provided?

Reply:
Good catch.
Although the self.file_system_type_version is ignored when `file_system_id` is provided, I improve the code to not provide any defaults if  `file_system_id` is provided